homeassistant/components/solaredge/coordinator.py

```diff
@@ -276,7 +276,7 @@ def update(self) -> None:
 
         for key, value in power_flow.items():
             if key in ["LOAD", "PV", "GRID", "STORAGE"]:
-                self.data[key] = value["currentPower"]
+                self.data[key] = value.get("currentPower", None)
```

Comment:
`None` is the default

```suggestion
                self.data[key] = value.get("currentPower")
```